src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BlockMerkleRootRule.cs

```diff
@@ -70,63 +73,57 @@ public static uint256 BlockMerkleRoot(Block block, out bool mutated)
         /// <param name="mutated"><c>true</c> if at least one leaf of the merkle tree has the same hash as any subtree. Otherwise: <c>false</c>.</param>
         public static uint256 ComputeMerkleRoot(List<uint256> leaves, out bool mutated)
         {
-            var branch = new List<uint256>();
-
             mutated = false;
-            if (leaves.Count == 0)
-                return uint256.Zero;
+            if (leaves.Count == 0) return uint256.Zero;
```

Comment:
I was telling @noescape the same thing. Try to avoid changes in code style for which we don't have rule. This leads to subjective changes because you like this way of writing the code more than what was there before, but if for example @noescape was about to edit this code later, he prefer the other way and he would change it back. So we would up in an infinite loop of changes. So the golden rule is, if there is no rule in the official code style policy, leave it as is.

But you don't need to fix it back here, it is just for the future.

Reply:
OK, thanks, I probably did not even do it on purpose... I'll keep that in mind